ffc-pay-alerts/notify/format-email-addresses.js

```diff
@@ -1,6 +1,9 @@
 const formatEmailAddresses = (emailAddresses) => {
-  const formatted = emailAddresses.replaceAll(' ', '')
-  return formatted.split(',')
+  if (typeof (emailAddresses) === 'string') {
+    // process.env.NOTIFY_EMAIL_ADDRESSES returns string, so splitting by comma to get array
+    emailAddresses = emailAddresses.split(',')
+  }
+  return emailAddresses.map(email => email.replaceAll(' ', ''))
 }
 
```

Comment:
Updated this as there was a scenario where when sending to multiple email addresses if we have two email addresses separated by commas and a space eg: `"simon.dunn1@defra.gov.uk, simon.dunn1@defra.gov.uk"`  they end up in an array like this `["simon.dunn1@defra.gov.uk", " simon.dunn1@defra.gov.uk"]` and notify rejects the second email because it has a space in front of it.

Therefore updated formatEmailAddress and associated tests to handle email addresses in string and arrray format and remove any rogue spaces.